community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/BuiltInProcedures.java

```diff
@@ -52,5 +52,8 @@ public void accept( Procedures procs ) throws ProcedureException
         // functionality - eg. things that apply across databases.
         procs.register( new ListProceduresProcedure( procedureName( "sys", "procedures" ) ) );
         procs.register( new ListComponentsProcedure( procedureName( "sys", "components" ), neo4jVersion ) );
+
+        // These are 'sys.auth'-namespaced procedures; these deal with authentication and authorization-oriented operations
+        procs.register( new AlterUserPasswordProcedure( procedureName( "sys", "auth", "changePassword" ) ) );
```

Comment:
We talked this over in the drivers and we'd like to not introduce nested namespaces right now, please change to `sys.changePassword`

We can pick this discussion up again once we add more `auth` related functions.